### PR TITLE
remove doc on 32/64 bit jvm env var for jenkins

### PR DESCRIPTION
**Pull request by gabemontero**
/assign @bmcelvee 

/assign @bparees

### CODE REVIEW

**Comment by bmcelvee** [comment]
LGTM - to which versions does this apply?

**Comment by gabemontero** [comment]
only 4.0 and beyond @bmcelvee

**Comment by gabemontero** [comment]
bump @bmcelvee 

were you waiting for any input from @bparees ?  ... we should be OK without it if need be

**Comment by bmcelvee** [comment]
Thanks Gabe! From Slack:

The content is good, though since it's for 4.x I need to bring forward the Jenkins content and open a new PR with the updates.

File: using_images/other_images/jenkins.adoc [hunk 1]
```diff
@@ -180,12 +180,6 @@ logging into Jenkins. To enable, set to `true`.
 The password for the `admin` user when using standard Jenkins authentication.
 Not applicable when `OPENSHIFT_ENABLE_OAUTH` is set to `true`.
 
-* `OPENSHIFT_JENKINS_JVM_ARCH`
```
Comment (bmcelvee) [inline using_images/other_images/jenkins.adoc]: Removed in https://github.com/openshift/openshift-docs/pull/14227.

**Comment by bmcelvee** [comment]
Closing - content brought forward/removed in https://github.com/openshift/openshift-docs/pull/14227.

**Pull request closed without merging**